.claude/skills/release-pr/SKILL.md

```
@@ -128,20 +124,30 @@ EOF
 7. **Discord 알림** — `#api-update` 채널에 변경사항 전송:
    - `~/.claude/env` 파일에서 `DH_API_UPDATE_DISCORD_WEBHOOK_URL` 로드
    - 파일이 없거나 변수가 없으면 → 안내 메시지 출력 후 skip (릴리스 자체는 정상 완료)
-   - PR Summary를 프론트엔드 팀원이 이해하기 쉬운 포맷으로 변환:
+   - PR Summary를 프론트엔드 팀원이 이해하기 쉬운 포맷으로 변환
+   - **Discord Markdown 포맷 규칙**:
+     - 들여쓰기 기반 포맷 사용 금지 (Discord에서 무시됨)
+     - `**볼드**`와 백틱 `` ` `` 코드블록 위주로 구성
+     - 섹션 구분에는 이모지 접두어 사용 (📌, ✅ 등)
+     - 빈 줄로 섹션 간 구분
 
 ```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)